content/en/blog/2021/zero-config-istio/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Zero Configuration Istio"
+description: Understanding the benefits Istio brings, even when no configuration is used.
+publishdate: 2021-02-25
+attribution: "John Howard (Google)"
+---
+
+Often times when a new user encounters Istio for the first time, they are overwhelmed by the vast feature
+set exposed by Istio. Unfortunately, this occasionally gives the impression that Istio is needlessly complex
```

Comment:
```suggestion
When a new user encounters Istio for the first time, they are sometimes overwhelmed by the vast feature
set it exposes. Unfortunately, this can give the impression that Istio is needlessly complex
```